app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/reducer.ts

```diff
@@ -0,0 +1,63 @@
+import { ISelectColumnsState, ReplicateSelect } from './types';
```

Comment:
This file needs a license

Reply:
ugh - i gotta look into a linter/ auto thing for us